tests/e2e-pw/playwright.config.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/**
+ * External dependencies
+ */
+import { defineConfig, devices } from '@playwright/test';
+
+const { BASE_URL } = process.env;
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig( {
+	testDir: './specs/',
+	/* Run tests in files in parallel */
+	fullyParallel: true,
```

Comment:
Shouldn't this be `false` since we're not running tests in parallel and limiting the number of workers to 1?

Reply:
💯 resolved in d8940c1